import-service/src/handlers/importProductsFile.test.js

```diff
@@ -0,0 +1,14 @@
+import AWS from 'aws-sdk-mock';
+import {S3} from "aws-sdk";
+
+describe('importProductsFile', () => {
+    it('should return signed url if getSignedUrl was called for puObject event', async () => {
```

Comment:
Here should be tested your implementation of **importProductsFile**, if you delete **import-service/src/handlers/importProductsFile.js** this test will still pass